src/content/guides/how-to-register-an-ethereum-account/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: How to "register" an Ethereum account
+description: A step-by-step guide on Ethereum account creation using a wallet.
+lang: en
+---
+
+# How to "register" an Ethereum account
+
+Anyone can create an Ethereum account for free with a specific type of app commonly known as a wallet. Wallets hold the keys that let you hold, send and receive crypto. You can also connect to projects on Ethereum that will let you trade NFTs, exchange coins, access play-to-earn games, unlock Defi and much more.
```

Comment:
```suggestion
Anyone can create an Ethereum account for free with a specific type of app commonly known as a wallet. Wallets hold the keys that let you hold, send and receive crypto. You can also connect to projects on Ethereum that will let you trade NFTs, exchange coins, access games, unlock Defi and much more.
```

I think we should be more inclusive of othe genres as well. Dark Forrest being an rts comes to mind here.